pupil-spa/src/app/question.service.ts

```diff
@@ -55,4 +54,22 @@ export class QuestionService {
 
     return currentQuestionNumber + 1;
   }
+
+  getConfig() {
+    return this.config;
+  }
+
+  reset() {
+    this.questions = null;
+    this.config = null;
+  }
+
+  initialise() {
```

Comment:
QuestionService is a singleton, and the state is kept throughout the lifetime.  So, we need to reload the questions on login and clear them on logout.